### DISCUSSION TITLE
New Beta -- Discussion on what is changing, improvements, etc.

### DISCUSSION THREAD

**Discussion by philipellisis** [PaloAltoNetworks/docusaurus-openapi-docs#522] [General]
I noticed that there are few new tags for the beta version. I was just curious what is the motivation for the new version and what can I expect to be changing. Are there performance improvements and new features I should be aware of? Any reason why I should wait to try the new Beta release? I noticed in [this discussion](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/discussions/408) and [this bug fix](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/issues/396) that beta fixes the tabs issue, but are there any other changes that are coming?

**Comment by sserrata** [comment]
Hi @philipellisis, great question. The biggest motivator for v2 is support for Docusaurus 2.3.0+, which required some breaking changes. In addition to that, we're taking the opportunity to optimize the UI/UX design, which will be a mix of some subtle/additive improvements and a few "bigger" design changes. We're still early phases and may solicit feedback from the community on design proposals.

## Support

As we develop v2 we will continue to support v1 (especially since we still use v1 in production). That said, any bug fixes introduced in v2 will be ported to v1 and vice versa, for the time being. The long term plan is to move to sunset v1 as more and more users upgrade to v2.3/2.4+, at which point v1 will be almost entirely community supported.

**Reply by gii-pan** [reply]
For the UI/UX design, maybe in the dark mode change the color of the background, because we can't see the difference between the site background and the tabs. 

![image](https://user-images.githubusercontent.com/64155751/231529327-47ecb6fa-8887-4f7b-9675-68e426a60a09.png)

**Comment by sserrata** [comment]
Here's a link to the v2.0.0 PR: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/pull/439

Preview URL: https://docusaurus-openapi-36b86--pr439-0r874qft.web.app/
